release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -63,6 +63,17 @@ User-defined tags for Microsoft Azure was previously introduced as Technology Pr
 [id="ocp-4-14-web-console"]
 === Web console
 
+[id="ocp-4-14-openshift-cli"]
+=== OpenShift CLI (oc)
+
+.Support multi-arch OCI local images for catalogs with oc-mirror
+
+With {product-title} {product-version}, oc-mirror supports multi-arch OCI local images for catalogs.
+
+OCI layouts consist of an `index.json` file that identifies the images held within them on disk. This `index.json` file can reference any number of single or multi-arch images. However, oc-mirror only references a single image at a time in a given OCI layout. The image stored in the OCI layout can be a single-arch image (that is, an image manifest) or a multi-arch image (that is, a manifest list).
```

Comment:
As per ISG, avoid using parentheses in general text. I am even wondering whether `that is ...` is required.

```suggestion
OCI layouts consist of an `index.json` file that identifies the images held within them on disk. This `index.json` file can reference any number of single or multi-arch images. However, oc-mirror only references a single image at a time in a given OCI layout. The image stored in the OCI layout can be a single-arch image, that is, an image manifest or a multi-arch image, that is, a manifest list.
```

Reply:
Ack.